src/react-apps/ux-editor/src/components/config/EditModalContent.tsx

```diff
@@ -377,14 +377,22 @@ export class EditModalContent extends React.Component<IEditModalContentProps, IE
 
     return (
       this.props.textResources.map((resource, index) => {
+        const option = this.truncate(resource.value);
         return (
           <option key={index} value={resource.id}>
```

Comment:
Would add a `title`-attribute here so that the user can see the full text as a tooltip.
```suggestion
          <option key={index} value={resource.id} title={resource.value}>
```